Source/CarthageKit/GitHub.swift

```diff
@@ -211,6 +211,9 @@ internal struct GitHubCredentials {
 
 				return nil
 			}
+			.catch { error in
+				return .single(nil)
```

Comment:
Yes. I wanted to ensure that we tried an unauthenticated request if we failed to load GitHub credentials.

You could definitely argue that this is the callers responsibility. I opted to put it here because it already returns an optional and we don't really care what the error is. IMO, it's hard to say what the best approach is because this method is only used in one place.